start_cerberus.py

```diff
@@ -165,6 +167,10 @@ def main(cfg):
                                            watch_namespaces_status, failed_nodes,
                                            failed_pods_components)
 
+            if iteration != 1 and crashed_restarted_pods:
+                logging.info("Pods that were crashed/restarted during the sleep: %s"
```

Comment:
Suggestion:  "Pods that were crashed/restarted during iteration %s : %s" % (iteration, crashed_restarted_pods)"